src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -65,14 +66,12 @@ public class MenuAndToolBarPresenter : AbstractMenuAndToolBarPresenter, IMenuAnd
       private readonly IActiveSubjectRetriever _activeSubjectRetriever;
       private bool _enabled;
       private SimulationState _simulationState;
-      private List<string> _activePIs;
-      private string _currentlyActivePI;
+      private readonly List<ParameterIdentification> _runningParameterIdentifications = new List<ParameterIdentification>();
```

Comment:
Let's not use id. LEt's use object and reference. Id is only for serialization 
ALso run resharper to see that the field could be readonly